### PR TITLE
Update Portuguese translation

### PR DESCRIPTION
**Pull request by hugok79**
Updated translations...
@ryonakano Happy new year!

### CODE REVIEW

File: po/extra/pt.po [hunk 1]
```diff
@@ -1,22 +1,22 @@
-# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
-# This file is distributed under the same license as the extra package.
-#
-# Hugo Carvalho <hugokarvalho@hotmail.com>, 2021.
+# Portuguese translations for com.github.ryonakano.reco package.
+# Copyright (C) 2019 THE com.github.ryonakano.reco'S COPYRIGHT HOLDER
```
Comment (ryonakano) [inline po/extra/pt.po:2]:
```suggestion
# Copyright (C) 2019-2022 THE com.github.ryonakano.reco'S COPYRIGHT HOLDER
```

**Review by ryonakano — COMMENTED** [review]
Happy new year too, @hugok79!🎍

Just one small comment and it's ready to go.

File: po/extra/pt.po (hunk 1 above)
Comment (hugok79) [reply po/extra/pt.po]: Done

**Review by ryonakano — APPROVED** [review]
Looks good, thank you as always! :tada:

**Pull request merged**